### PR TITLE
pgrep: add more info link

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]

File: pages/common/pgrep.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # pgrep
 
 > Find or signal process by name.
```
Comment (marchersimon) [inline pages/common/pgrep.md:3]:
```suggestion
> Find or signal processes by name.
```

File: pages/common/pgrep.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 - Return PIDs of any running processes with a matching command string:
 
 `pgrep {{process_name}}`
 
 - Search full command line with parameters instead of just the process name:
 
-`pgrep -f "{{process_name}} {{parameter}}"`
+`pgrep --full "{{process_name}} {{parameter}}"`
 
 - Search for process run by a specific user:
```
Comment (marchersimon) [inline pages/common/pgrep.md:14]:
```suggestion
- Search for processes run by a specific user:
```

File: pages/common/pgrep.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
 # pgrep
 
 > Find or signal process by name.
+> More information: <https://www.man7.org/linux/man-pages/man1/pkill.1.html>.
 
 - Return PIDs of any running processes with a matching command string:
 
 `pgrep {{process_name}}`
 
 - Search full command line with parameters instead of just the process name:
```
Comment (marchersimon) [inline pages/common/pgrep.md:10]:
```suggestion
- Search for processes, specifying its command line options:
```
Maybe this will read a little better?

**Review by marchersimon — CHANGES_REQUESTED** [review]

File: pages/common/pgrep.md (hunk 3 above)
Comment (navarroaxel) [reply pages/common/pgrep.md:10]: I prefer something like: `Search for a process name including the command line options`, or similar.

Comment (marchersimon) [reply pages/common/pgrep.md:10]: Sure, but I think we should still use the plural, since we search for all matching processes.

Comment (marchersimon) [reply pages/common/pgrep.md:10]:
```suggestion
- Search for processes including their command line options:
```

**Review by marchersimon — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**